api-catalog-ui/frontend/package.json

```diff
@@ -43,13 +43,13 @@
         "start": "cross-env HTTPS=true react-scripts start",
         "start:dev": "concurrently -c blue,green -n Backend,UI \"npm run start:mock\" \"cross-env REACT_APP_GATEWAY_URL=http://localhost:8000 npm run start\"",
         "start:mock": "cross-env REACT_APP_GATEWAY_URL=http://localhost:8000 nodemon mock/server.js",
-        "build": "pnpm install && react-scripts build",
+        "build": "react-scripts build",
```

Comment:
Why did you remove the pnpm install?

Reply:
Because it was happening multiple times. 
I moved it to the installation step.

Reply:
And as it turned out it wasn't necessary to keep the usage of the build cache. 
Only the pnpm install needs to be run before the build and test

Reply:
Oh right, I've seen now. It makes sense